core/server/data/schema/schema.js

```diff
@@ -296,6 +296,11 @@ module.exports = {
         id: {type: 'string', maxlength: 24, nullable: false, primary: true},
         event: {type: 'string', maxlength: 50, nullable: false, validations: {isLowercase: true}},
         target_url: {type: 'string', maxlength: 2000, nullable: false},
+        name: {type: 'string', maxlength: 191, nullable: true},
+        secret: {type: 'string', maxlength: 191, nullable: true},
```

Comment:
`secret` is entered by a user which can either be a passphrase sent along with request(like Prismic) or can be used to create a hash(like Github) to validate request came from Ghost. Not used right now, its still todo on how we should ideally use it, but something we discussed UI would allow to enter already.